src/components/accountTransactions/accountTransactions.css

```diff
@@ -7,9 +7,15 @@
 
 .wrapper {
   min-width: 100%;
+  box-shadow: 0px 22px 22px 0px rgba(0, 0, 0, 0.22);
```

Comment:
We have `var(--normal-shadow)` defined in `variables.css` that you can use here.

Reply:
oh thanks!